app/src/main/java/com/swent/suddenbump/ui/authentication/SignUp.kt

```diff
@@ -166,66 +166,45 @@ fun SignUpScreen(navigationActions: NavigationActions, userViewModel: UserViewMo
                         }
                   }
 
+              val textFieldColors =
+                  TextFieldDefaults.colors(
+                      focusedTextColor = Color.White,
+                      unfocusedTextColor = Color.White,
+                      disabledTextColor = Color.Gray,
+                      errorTextColor = Color.Red,
+                      cursorColor = Color.White,
+                      errorCursorColor = Color.Red,
+                      focusedIndicatorColor = Color.White,
+                      unfocusedIndicatorColor = Color.Gray,
+                      disabledIndicatorColor = Color.LightGray,
+                      errorIndicatorColor = Color.Red,
+                      focusedLabelColor = Color.White,
+                      unfocusedLabelColor = Color.Gray,
+                      disabledLabelColor = Color.LightGray,
+                      errorLabelColor = Color.Red,
+                      focusedPlaceholderColor = Color.White,
+                      unfocusedPlaceholderColor = Color.Gray,
+                      disabledPlaceholderColor = Color.LightGray,
+                      errorPlaceholderColor = Color.Red,
+                      focusedContainerColor = Color.Black,
+                      unfocusedContainerColor = Color.Black,
+                      disabledContainerColor = Color.DarkGray,
+                      errorContainerColor = Color.Black)
+
```

Comment:
Very good practice here: instead of recalling these colors every time, it's much better to have a textFieldColors set-up